bin/migrate.php

```diff
@@ -27,27 +29,54 @@
 $io = new SymfonyStyle(new ArgvInput(), new ConsoleOutput());
 
 $inputDefinition = new InputDefinition([
-    new InputArgument('asset-family-code', InputArgument::REQUIRED),
-    new InputArgument('pim-path', InputArgument::REQUIRED)
+    new InputArgument('asset_family_code', InputArgument::REQUIRED),
+    new InputArgument('pim_path', InputArgument::REQUIRED),
+    new InputOption('--help', '-h', InputOption::VALUE_NONE, 'Display this help message'),
 ]);
 
+if (count(array_intersect($_SERVER['argv'], ['-h', '--help']))) {
+    $io->writeln(sprintf("<comment>Description:</comment>
+  Migrate an asset family in full-automatic scenario
```

Comment:
Link to doc would be cool!

Reply:
There are already 4 links in the README, isn't it OK?

Reply:
NO ITS NOT OK NOT GTM PIERRE YOUR WORK IS SO NOT LINKABLE

Reply:
(ok no link)